lib/home/users/permissions.dart

```diff
@@ -20,64 +23,52 @@ class RadioGroup extends StatefulWidget {
   final Map<String, dynamic> user;
 
   @override
-  _RadioGroupState createState() => _RadioGroupState();
+  State<RadioGroup> createState() => _RadioGroupState();
 }
 
 class _RadioGroupState extends State<RadioGroup> {
-  late String _selectedRadio;
-
-  @override
-  void initState() {
-    super.initState();
-    _selectedRadio = widget.user['type'];
-  }
+  late UserType _selectedRadio = UserType.fromJson(widget.user);
```

Comment:
In general, `late` variables can be initialized right away instead of needing to call `initState()`.

Also, I thought it'd be nice to use the existing `UserType` enum for this.